.github/workflows/build.yml

```diff
@@ -80,6 +84,10 @@ jobs:
           repository: ${{ inputs.binary_repo }}
           ref: ${{ inputs.ref }}
           path: repo
+      - if: ${{ inputs.patch }}
+        run: |
+          [[ ! -f ./patches/${{ inputs.binary_repo }}.patch ]] && exit 1
+          cd ./repo && git apply -3 ../patches/${{ inputs.binary_repo }}.patch
```

Comment:
Shall we build both the `non-patched` version and the `patched` version?

Reply:
Given that users specify the version in their use of `download`, if there's ever a day or two where the latest build is breaking, so be it. Better to keep things simple imo.